e2e_playwright/multipage_apps/mpa_configure_sidebar_test.py

```diff
@@ -100,7 +100,7 @@ def test_page_link_href(
     """Test that page link href set properly."""
     page_links = app.get_by_test_id("stPageLink-NavLink")
 
-    expect(page_links.nth(0)).to_have_attribute("href", "mpa_configure_sidebar")
+    expect(page_links.nth(0)).to_have_attribute("href", "")
```

Comment:
I guess in MPA v1, the first page was the default page and had the correct url from the file, but it also works as the default with no path name. So this seems like a reasonable expectation.